inc/config.class.php

```diff
@@ -3544,4 +3549,32 @@ public static final function generateUuid($type) {
       self::setConfigurationValues('core', [$type . '_uuid' => $uuid]);
       return $uuid;
    }
+
+   /**
+    * Get a file system adapter from configuration
+    *
+    * @return FilesystemAdapter
+    */
+   public static function getFsAdapter(): FilesystemAdapter {
```

Comment:
If we want this to be reusable across all VAR directories, I suggest to:
1. Add a mandatory parameter `string $directory`. This parameter will be used as the path for `LocalFilesystemAdapter`, but may also be used in other adapters to identify which "virtual" directory we want to use.
2. Do not put result into a global variable, as we have many var directories and so we will need too many global variables. Maybe we can use a private array (`[$dir1 => $adapter1, $dir2 => $adapter2, ...]`) if wee nedd reusable data.
3. Why not directly returning an instance of `League\Flysystem\Filesystem` ?

Reply:
1 and 3 changed. FOr 2, I now rely on a path based entry in the configuration array